src/core/errorHandler.js

```diff
@@ -31,41 +36,57 @@ export class ErrorHandler {
 
 class ErrorHandlerComponent extends React.Component {
   static propTypes = {
+    autoRenderErrors: PropTypes.boolean,
     dispatch: PropTypes.func,
     error: PropTypes.object,
     errorHandlerId: PropTypes.string,
     WrappedComponent: PropTypes.object,
   }
 
+  static defaultProps = {
+    autoRenderErrors: true,
+  }
+
   render() {
     const {
+      autoRenderErrors,
       WrappedComponent,
       errorHandlerId,
       dispatch,
       error,
       ...props
     } = this.props;
-    const errorHandler = new ErrorHandler({ id: errorHandlerId, dispatch });
-    const wrappedOutput = (
-      <WrappedComponent errorHandler={errorHandler} {...props} />
-    );
 
+    let errorContent;
     if (error) {
+      errorContent = (
+        <ul className="ErrorHandler-list">
+          {error.messages.map((msg) => <li>{msg}</li>)}
+        </ul>
+      );
+    }
+
+    const errorHandler = new ErrorHandler({
+      errorContent,
+      id: errorHandlerId,
+      dispatch,
+    });
+    const allProps = { ...props, errorHandler };
+
+    if (error && autoRenderErrors) {
```

Comment:
I think it makes more sense to split out the rendering and the display than to pass something to this one entry point to tell it to only do one or the other.

You could pull the `connect` bits out into a `withErrorHandler` function which could be used by `withErrorHandling` and this component instead of telling the HOC to skip rendering errors.